src/components/Select/components/SelectControl/SelectControl.scss

```diff
@@ -97,6 +113,16 @@ $blockButton: '.#{variables.$ns-new}select-control__button';
 }
 
 #{$block} {
+    --_--select-disabled-text-color: var(--g-color-text-hint);
```

Comment:
No need to name these vars so descriptive, they're private anyway. Let's shorten these to `--_--text-color`, `--_--background-color` and so on